firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -427,6 +427,51 @@ public void testQueriesCanUseArrayContainsFilters() {
     // much of anything else interesting to test.
   }
 
+  @Test
+  public void testQueriesCanUseInFilters() {
```

Comment:
These tests only work against the emulator, right?  So we need to figure out a way to guard / disable them for now?

Reply:
Yeah... Still trying to figure out how to even run the tests against the emulator, but i'll update this after.